src/Place/ReadModel/JSONLD/PlaceLDProjector.php

```diff
@@ -292,7 +308,23 @@ function ($term) {
         $placeLd->terms = $terms;
 
         $this->repository->save($document->withBody($placeLd));
+    }
 
+    /**
+     * @param GeoCoordinatesUpdated $geoCoordinatesUpdated
+     */
+    protected function applyGeoCoordinatesUpdated(GeoCoordinatesUpdated $geoCoordinatesUpdated)
+    {
+        $document = $this->loadPlaceDocumentFromRepository($geoCoordinatesUpdated);
+
+        $placeLd = $document->getBody();
+
+        $placeLd->geo = (object) [
+            'latitude' => $geoCoordinatesUpdated->getCoordinates()->getLatitude()->toDouble(),
+            'longitude' => $geoCoordinatesUpdated->getCoordinates()->getLongitude()->toDouble(),
```

Comment:
Ik denk toch dat we beter een transformer gebruiken op de search service. We gaan dat ook nodig hebben om location.address en location.geo te verhuizen naar de root van het document op events. Anders wordt het moeilijk zoeken op zowel places als events die op een bepaalde locatie (geografisch) doorgaan.